templates/docs/examples/patterns/chip/with-dismiss.html

```diff
@@ -7,8 +7,178 @@
 <div class="p-chip">
   <span class="p-chip__lead">Owner</span>
   <span class="p-chip__value">Bob</span>
+  <span class="p-chip__badge">0</span>
   <button class="p-chip__dismiss">
     <i class="p-icon--close">Dismiss</i>
   </button>
 </div>
+
+<div class="p-chip">
```

Comment:
I'm not sure if that's the right example to do that. It is supposed to be about dismiss functionality of the chip, not all the colors, badges, etc.

Keep this example as it was please, add separate (minimal one, to be embedded in docs, with just one example per color), and if needed, another one, not to be embedded, with exploration of any combinations of colors, dismiss, and other features.

Reply:
agree, thst's how it was in the original pr, will move

Reply:
fixed